distributive/distributive-docker/distributive-docker.json

```diff
@@ -0,0 +1,61 @@
+{
+    "Name": "docker health check",
+    "Notes": "For use with Microservices Infrastructure",
+    "Checklist" : [
+        {
+            "Name": "device-mapper-libs installation check",
+            "ID": "installed",
+            "Parameters": ["device-mapper-libs"]
+        },
+        {
+            "Name": "docker installation check",
+            "ID": "installed",
+            "Parameters": ["docker"]
+        },
+        {
+            "Name": "docker dns configuration file",
+            "ID": "file",
+            "Parameters": ["/etc/sysconfig/docker-network"]
+        },
+        {
+            "Name": "docker systemd service enabled",
+            "ID": "systemctlLoaded",
+            "Parameters": ["docker.service"]
+        },
+        {
+            "Name": "docker collectd plugin dependency - py-dateutil",
```

Comment:
We've had an issue before with a failure of collectd causing issues with docker. The docker collectd checks should be separate from docker itself. 

This issue is also present in the zookeeper checks.